### PR TITLE
1206-Phone-Number-Error-and-Hint-Text-Post-Circle-PR

### PR DESCRIPTION
**Pull request by Dmac26**
﻿## Summary
Addresses Issue # 1165
This PR is an updated version of the original 1165 PR, which needed several styling updates to account for mobile views and to clean up some syntax. I was unable to directly update the original PR as I did not manage my work flow properly and my branch was significantly behind the dev branch.

## This pull request is ready to merge when...
- [x] Feature branch is starts with the issue number
- [x] Is connected to its original issue via zenhub 👇
- [x] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Jenkins)
- [ ] This code has been reviewed by someone other than the original author

### CODE REVIEW

**Comment by briandavidson** [comment]
image

Prior, the user was unable to enter more than the allowed 10 characters. Let's revert back to that functionality, and we should still be able to address the the concerns of the original issue.

File: server/package.json [hunk 1]
```diff
@@ -38,10 +38,10 @@
     "passport-local": "^1.0.0",
     "passport-saml": "^1.1.0",
     "passport-strategy": "^1.0.0",
-    "pg": "^6.0.0",
+    "pg": "^6.4.2",
     "request": "^2.88.0",
     "request-promise-native": "^1.0.5",
-    "sequelize": "^5.15.1",
+    "sequelize": "^5.21.10",
```
Comment (briandavidson) [inline server/package.json:44]: We shouldn't need to update the versions of any packages for this issue. It's most likely harmless, but for concision, clarity, and keeping within the scope of the issue, we should isolate the changes to just what's necessary. Please revert the changes to this file, and omit `package-lock.json` from the PR as well.

Comment (Dmac26) [reply server/package.json]: @briandavidson - I fixed the problem with the allowable number of digits for the phone number field, I was able to successfully revert both the package.json and the package-lock.json files, but I'm not sure how to omit it from the PR. Can I just go in and delete it? I was worried that would remove it from Dev once we merge.

Comment (Dmac26) [reply server/package.json]: @briandavidson - I was able to omit both .json files since there is no need to make any changes to them for this issue.

**Comment by Dmac26** [comment]
@mtlaney @briandavidson - This one should be good to go too!
